supreme_court_predictions/statistics/service.py

```diff
@@ -3,6 +3,9 @@
 """
 
 from supreme_court_predictions.processing.datacleaner import DataCleaner
+from supreme_court_predictions.processing.token_aggregation import (
+    TokenAggregations,
+)
```

Comment:
Long line is lonnnnnng.
![image](https://user-images.githubusercontent.com/5885605/236367174-98d9742c-bd38-4ceb-89d8-dd86fa53055b.png)